.github/workflows/cli-ci.yml

```
@@ -0,0 +1,89 @@
+name: cli/ubi CI
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'cli/go.mod'
+      - 'cli/ubi.go'
```

You can use a pattern instead of adding each file one by one.
```suggestion
      - 'cli/**'
```

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths